### PR TITLE
Add Starcraft Launcher.app Latest

### PR DESCRIPTION
**Pull request by mikesplain**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Review by miccal — APPROVED** [review]

File: Casks/starcraft.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # battle.net was verified as official when first introduced to the cask
+  url 'https://www.battle.net/download/getInstallerForGame?os=MAC&version=LIVE&gameProgram=STARCRAFT'
+  name 'Starcraft'
+  homepage 'https://starcraft.com/'
+
+  depends_on macos: '>= :el_capitan'
+
+  installer manual: 'StarCraft Launcher.app'
+
+  uninstall delete: '/Applications/Starcraft'
```
Comment (vitorgalvao) [inline Casks/starcraft.rb]: Are you sure moving this app is the only thing the app installer does?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by zeph** [comment]
ok nevermind, I was opening an identical pull request (good job with the checklist 😅  )

File: Casks/starcraft.rb (hunk 1 above)
Comment (mikesplain) [inline Casks/starcraft.rb]: Yeah the installer puts everything in this directory. I added another directory to the zap list as well.  This is a similar installer to the [battle-net](https://github.com/caskroom/homebrew-cask/blob/master/Casks/battle-net.rb) cask.

Comment (zeph) [inline Casks/starcraft.rb]: WARNING: the C of StarCraft, in the folder name, is capital
```bash
$ ls /Applications/StarCraft
Blizzard Error.app	License.txt		StarDat.mpq
BrooDat.mpq		Maps			StarEditHelp
BroodWar.mpq		Release Notes.txt	Starcraft.mpq
Data			StarCraft Launcher.app	SystemSurvey
Launcher.db		StarCraft Manual.pdf	patch_rt.mpq
License.html		StarCraft.app```

Comment (mikesplain) [inline Casks/starcraft.rb]: Good eye, pushing a fix now.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**